packages/block-library/src/spacer/edit.js

```diff
@@ -197,6 +254,18 @@ const SpacerEdit = ( {
 				width: '72px',
 			} );
 		}
+		if ( isFlexLayout && ! flexSize ) {
+			setAttributes( {
+				style: {
+					...blockStyle,
+					layout: {
+						...layout,
+						flexSize: width || '72px',
```

Comment:
Does this need to account for `height` and/or the spacing presets values? When dragging into a Stack variation, it looks like it defaults to the `72px` value:

![2023-04-12 16 48 11](https://user-images.githubusercontent.com/14988353/231374667-757ad173-5589-49ab-8468-408431e95bc4.gif)

Reply:
Ah yes, well spotted! I've fixed the logic in the effect now and the problem with fill state should be addressed.